tests/js/roots/multiterm/index.rst

```diff
@@ -0,0 +1,4 @@
+Main Page
+=========
+
+Welcome to the... main page!
```

Comment:
I would personally use this opportunity to add a bit of context about what this file tests.